boards/imix/src/main.rs

```diff
@@ -117,7 +117,7 @@ const PAN_ID: u16 = 0xABCD;
 const FAULT_RESPONSE: kernel::procs::FaultResponse = kernel::procs::FaultResponse::Panic;
 
 #[link_section = ".app_memory"]
-static mut APP_MEMORY: [u8; 16384] = [0; 16384];
+static mut APP_MEMORY: [u8; 32768] = [0; 32768];
```

Comment:
Note: this is only peripherally related but didn't want to do a separate PR. It was increased so that imix can run some of the userspace tests which are designed for Hail, which allocates more RAM to apps.